css-two-column-layout/styles.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.row {
+  display: flex;
+}
+
+.center-column {
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.column-one {
+  margin-left: 250px;
+  margin-right: 50px;
+}
+
+.column-two {
+  margin-right: 250px;
+  margin-left: 50px;
+}
```

Comment:
This is a _lot_ of margin. You should try to do this without such a large measurement, you might try `justify-content: center` on the `row` for these items, and add just a bit of `padding` to each column in this row to put space between them